go/processor/floria/calls.go

```diff
@@ -16,62 +16,70 @@ import (
 	"github.com/Fantom-foundation/Tosca/go/tosca"
 )
 
-func call(interpreter tosca.Interpreter, transaction tosca.Transaction, context tosca.TransactionContext, gas tosca.Gas) (tosca.Result, error) {
+func (r runContext) Call(kind tosca.CallKind, parameters tosca.CallParameters) (tosca.CallResult, error) {
```

Comment:
It is surprising that the `runContext` is defined in the `processor.go` but its main only member function `Call` is defined here. I think the type definition should be moved here and the file may be renamed from `call.go` to `run.go` or `run_context.go`. Wdyt?

Reply:
Right, I moved it to the calls file and renamed it to run_context.go